pkg/apis/gateway/v1alpha1/types.go

```diff
@@ -42,17 +42,15 @@ type GatewayList struct {
 
 // GatewaySpec represents gateway specifications
 type GatewaySpec struct {
+
 	// Image is the gateway processor image
 	Image string `json:"image" protobuf:"bytes,1,opt,name=image"`
```

Comment:
what's your thought about using a `corev1.Container` definition instead of Image, ImagePullPolicy, Affinity and such?